nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -796,6 +796,42 @@ in
       };
     };
 
+    virtualisation.tpm = {
```

Comment:
Seems odd to have this in `qemu-vm.nix` when it requires the test driver. Either `test-instrumentation.nix` or an implementation that isn't dependent on the driver seems preferable (with the latter seeming fairly impractical).

Reply:
Good point.

Reply:
A driver-less implementation could be provided by modifying the QEMU run script and adding a parallel program to run `swtpm` actually.